src/bits/bit_array.rs

```diff
@@ -296,6 +303,24 @@ macro_rules! bit_array_impl {
 
                     assert_eq!(a, $name::deserialize(&buf).unwrap());
                 }
+
+                proptest! {
+                   #[test]
+                   fn identities(a in 2_u128..1 << $name::BITS - 1) {
```

Comment:
I'm not clear on the operator precedence rules here.  Which did you mean?

```suggestion
                   fn identities(a in 2_u128..((1 << $name::BITS) - 1)) {
```

or 

```suggestion
                   fn identities(a in 2_u128..(1 << ($name::BITS - 1))) {
```

Either way, this is a bit of an odd test.  You are testing all values that aren't 1 or 0 and then checking operations that only affect the last bit.  That's not going to be of much use.